app/src/main/java/com/brandon/forzenbook/network/ForzenApiService.kt

```diff
@@ -8,12 +8,14 @@ interface ForzenApiService {
 
     @GET(LOGIN_ENDPOINT)
     suspend fun getCode(
-        @Query("email") email: String
+        @Query(EMAIL) email: String
     ): Response<Unit>
 
+    @FormUrlEncoded
     @POST(LOGIN_ENDPOINT)
     suspend fun login(
```

Comment:
Maybe this should be something like "requestEmailCode"

Reply:
Will include in FA-89 since modules don't play nice with git.